fda-model/metrics/ScenarioResults.cs

```diff
@@ -41,52 +41,60 @@ public ScenarioResults(int year)
         public List<string> GetAssetCategories()
         {
             List<string> assetCats = new List<string>();
-            foreach (IContainImpactAreaScenarioResults containImpactAreaScenarioResults in _resultsList)
+            if (_resultsList.Count != 0)
```

Comment:
High cognitive complexity here. Multiple foreach loops and if statements. Can we move some of this into its own method maybe?